examples/code-splitting-native-import-context/README.md

```diff
@@ -0,0 +1,394 @@
+# example.js
+
+This example illustrates how to leverage the `import()` syntax to create ContextModules which are separated into separate chunks for each module in the `./templates` folder.
+
+``` javascript
+
+async function getTemplate(templateName) {
+	try {
+		let template = await import(`./templates/${templateName}`);
+		console.log(template);
+	} catch(err) {
+		console.error("template error");
+		return new Error(err);
+	}
+}
+
+getTemplate("a");
+getTemplate("b");
+getTemplate("c");
+```
+
+# templates/
+
+* a.js
+* b.js
+* c.js
+
+All templates are of this pattern:
+
+``` javascript
+module.exports = function() {
+	return "This text was generated by template X";
+}
```

Comment:
As far I know it's undefined so I decided for a behavior similar to `import * as x from ...`.

``` js
// fs.js
exports.readFile = function() {}
```

``` js
import("./fs").then(fs => a.readFile())
```

``` js
import * as fs from "./fs";
fs.readFile();
```

The way babel does it is just complex and weird:

``` js
import("./fs").then(fs => fs.default.readFile())
```

``` js
import * as fs from "./fs";
fs.default.readFile();
```

And it requires wrappers that increase bundle size.

---

As the spec doesn't define a behavior here I'm free to do whatever I want. It's rar anyway.